src/lib.rs

```diff
@@ -126,6 +128,9 @@ impl<S: Store> fmt::Display for VMError<S> {
             VMError::UnknownContract => write!(f, "Unknown Contract")?,
             VMError::InvalidWASMModule => write!(f, "Invalid WASM module")?,
             VMError::StoreError(e) => write!(f, "Store error {:?}", e)?,
+            VMError::ABICallExecError => {
+                write!(f, "Error during ABI call execution")?
+            }
```

Comment:
Don't add new error like that we don't have them in the other ABI and this will be refactored anyway the next sprint. If you have a result you can have an `expect` and panic, but it depends case by case. Since I cannot see in this PR the usage of this error, I cannot tell what should be done.

Reply:
I just want to have that so that when we print the error we know this is related to ABI execution stuff that we just added